apps/system/js/bluetooth_transfer.js

```diff
@@ -43,18 +36,28 @@ var BluetoothTransfer = {
     );
 
     // Listen to 'bluetooth-opp-transfer-start' from bluetooth.js
-    window.addEventListener('bluetooth-opp-transfer-start',
-      this.onUpdateProgress.bind(this, 'start')
-    );
+    window.addEventListener('bluetooth-opp-transfer-start', this);
 
     navigator.mozSetMessageHandler('bluetooth-opp-update-progress',
-      this.onUpdateProgress.bind(this, 'progress')
+      this._onUpdateProgress.bind(this, 'progress')
     );
 
     // Listen to 'bluetooth-opp-transfer-complete' from bluetooth.js
-    window.addEventListener('bluetooth-opp-transfer-complete',
-      this.onTransferComplete.bind(this)
-    );
+    window.addEventListener('bluetooth-opp-transfer-complete', this);
+  },
+
+  handleEvent: function bt_handleEvent(evt) {
+    switch(evt.type) {
+      case 'iac-bluetoothTransfercomms':
+        this._onFilesSending();
```

Comment:
As above comment, missed to pass parameter to `this._onFilesSending()` function.